spec/models/wiky_spec.rb

```diff
@@ -8,7 +8,7 @@
   end 
   context "create" do 
     it "wiky defaults to public " do 
-      wiky = Wiky.create
+      wiky = create(:wiky)
```

Comment:
By using FactoryGirl here, we're no longer testing that wiky defaults to public in the core application without FactoryGirl.

```ruby
wiky = Wiky.create # No dependency on FactoryGirl, good for testing defaults
wiky = create(:wiky) # Dependent on FactoryGirl, bad for testing defaults
```